resources/views/themes/default1/common/dashboard.blade.php

```diff
@@ -89,6 +89,22 @@
               <i class="fa fa-arrow-circle-right"></i></a>
              </div>
         </div>
+
+         <div class="col-lg-3 col-xs-6">
+          <!-- small box -->
+          <div class="small-box bg-yellow">
+            <div class="inner">
+              <h4>Subscriptions(Last 30 days)</h4>
+              <span>Total: &nbsp;  {{$getLast30DaysInstallation['total_subscription']}}</span><br/>
+              <span>Inactive: &nbsp;  {{$getLast30DaysInstallation['inactive_subscription']}}</span><br/>
```

Comment:
instead of inactive, you can say "not installed"

Reply:
Fixed